src/scores/categorical/risk_matrix_impl.py

```diff
@@ -0,0 +1,498 @@
+"""
+This module contains methods which for related to the risk matrix score.
```

Comment:
```suggestion
This module contains methods which are related to the risk matrix score.
```
Minor typo. Should this be "methods which are related" or "methods related" or something similar?